DSharpPlus.Commands/Trees/CommandParameterBuilder.cs

```diff
@@ -39,7 +39,7 @@ public CommandParameterBuilder WithDescription(string? description)
     public CommandParameterBuilder WithType(Type type)
     {
         this.Type = type;
-        if (type.IsEnum && !this.Attributes.Any(attribute => attribute is SlashChoiceProviderAttribute))
+        if (type.IsEnum && !this.Attributes.Any(attribute => attribute is SlashChoiceProviderAttribute or SlashAutoCompleteProviderAttribute))
```

Comment:
Perhaps `this.Attributes.All(attribute => attribute is not SlashChoiceProviderAttribute and not SlashAutoCompleteProviderAttribute)`?